Doc/library/test.rst

```diff
@@ -1691,6 +1691,21 @@ The :mod:`test.support.warnings_helper` module provides support for warnings tes
 .. versionadded:: 3.10
 
 
+.. function:: ignore_warnings(*, category)
+
+   A decorator to suppress warnings. *category* is a subclass of
+   :exc:`Warning` class. It is approximately equivalent to calling
+   :func:`warnings.catch_warnings()` with :meth:`warnings.simplefilter`
+   set to ``ignore``. If you want to suppress some specific known warnings
+   in a test case, use code like the following::
```

Comment:
```suggestion
   Suppress warnings that are instances of *category*,
   which must be :exc:`Warning` or a subclass.
   Roughly equivalent to :func:`warnings.catch_warnings`
   with :meth:`warnings.simplefilter('ignore', category=category) <warnings.simplefilter>`.
   For example::
```

* Rewrite more clearly and concisely
* Clarify explicit what *category* does
* Avoid implying that *category* must be a _strict_ subclass of `Warning`
* State what it is equivalent to more explicitly.
* Fix some minor grammar/phrasing issues I missed the first time

Reply:
Looks good to me. @CharlieZhao95 can you apply CAM's suggestion?

Reply:
> Looks good to me. @CharlieZhao95 can you apply CAM's suggestion?

Sure. Thanks to everyone for the careful review and suggestions 😄

Reply:
Thanks for your contribution!